js/resourceInfo.js

```diff
@@ -27,45 +27,66 @@ var licences = {
     "CC-BY-SA-3.0": { "name": "CC BY-SA 3.0", "url": "https://creativecommons.org/licenses/by-sa/3.0/" },
     "CC-BY-NC-ND-3.0": { "name": "CC BY-NC-ND 3.0", "url": "https://creativecommons.org/licenses/by-nc-nd/3.0/" },
     "OFL": { "name": "OFL", "url": "http://scripts.sil.org/OFL" },
+    "Freepik": { "name": "Freepik terms of use", "url": "https://www.freepikcompany.com/legal#nav-freepik" }
```

Comment:
There's a little mess in case of the licensing and attribution.
As far as I understand it should be:
`"Flaticon": { "name": "Flaticon terms of use", "url": "https://www.freepikcompany.com/legal#nav-flaticon" }`

It's only a coincidence that the icon you used is created by the same company that runs the whole flaticon website (Freepik). You can check this here:
![obraz](https://user-images.githubusercontent.com/21989150/87781614-47b1aa80-c831-11ea-8b5b-ae397f419b71.png)

This is from the license summary on the image site. It redirects to the page I've provided.

Reply:
Done